scipy/stats/tests/test_resampling.py

```diff
@@ -940,9 +940,11 @@ def stat(x, y, axis):
                              random_state='herring')
 
     # -- Test Parameters -- #
+    @pytest.mark.parametrize('random_state', [np.random.RandomState,
+                                              np.random.default_rng])
```

Comment:
It's important to test with both `RandomState` and `default_rng` because we use them in different ways.